src/Umbraco.Web/umbraco.presentation/umbraco/preview/PreviewContent.cs

```diff
@@ -11,11 +12,57 @@
 
 namespace umbraco.presentation.preview
 {
-    //TODO : Migrate this to a new API!
+    public enum PreviewMode
+    {
+        Unknown = 0, // default value
+        Vintage,

```

Comment:
I think `Legacy` and `Default` might be better

Reply:
Yup, these names were clearly "in progress" - Legacy and Default sounds good assuming we make the new engine the default. Will change.

Reply:
have pushed changes